GovUkDesignSystem/GovUkDesignSystemComponents/ItemSet.cshtml

```diff
@@ -9,23 +10,22 @@
 
     Func<object, object> innerHtml =
         @<text>
-            <div class="govuk-form-group @( Model.FormGroup?.Classes) @( Html.Raw(Model.ErrorMessage != null ? "govuk-form-group--error" : ""))">
+            <div class="govuk-form-group @( Model.FormGroup?.Classes) @(Model.ErrorMessage != null ? "govuk-form-group--error" : "")">
                 @{
-                    if (Model.Hint != null)
-                    {
-                        Model.Hint.Id = hintId;
-                        await Html.RenderPartialAsync("/GovUkDesignSystemComponents/Hint.cshtml", Model.Hint);
-                    }
-                    if (Model.ErrorMessage != null)
-                    {
-                        Model.ErrorMessage.Id = errorId;
-                        await Html.RenderPartialAsync("/GovUkDesignSystemComponents/ErrorMessage.cshtml", Model.ErrorMessage);
-                    }
+                if (Model.Hint != null)
+                {
+                    Model.Hint.Id = hintId;
+                    await Html.RenderPartialAsync("/GovUkDesignSystemComponents/Hint.cshtml", Model.Hint);
                 }
-
-                <div class="@( Model.StyleNamePrefix) @( Model.Classes) @( Html.Raw(isConditional ? $"{Model.StyleNamePrefix}--conditional" : ""))"
-                     @( Html.Raw(isConditional ? $"data-module={Model.StyleNamePrefix}" : ""))
-                     @( Html.Raw(Model.Attributes != null ? string.Join(" ", Model.Attributes.Select(a => a.Key + "=\"" + a.Value + "\"")) : ""))>
+                if (Model.ErrorMessage != null)
+                {
+                    Model.ErrorMessage.Id = errorId;
+                    await Html.RenderPartialAsync("/GovUkDesignSystemComponents/ErrorMessage.cshtml", Model.ErrorMessage);
+                }
+                }
+                <div class="@( Model.StyleNamePrefix) @( Model.Classes) @(isConditional ? $"{Model.StyleNamePrefix}--conditional" : "")" 
+                     @(isConditional ? $"data-module={Model.StyleNamePrefix}" : "")
+                     @(Html.Raw(Model.Attributes.ToTagAttributes()))>
```

Comment:
Correct, these attributes are the only place where we actually have to use Html.Raw otherwise you end up with things like:
`attr-name=&quot;attr-value&quot;`